runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsConfig.java

```diff
@@ -478,9 +478,57 @@ default boolean notifyOnlyOnSkulledPlayers()
 		return false;
 	}
 
-	@Units(Units.TICKS)
 	@ConfigItem(
 			position = 33,
+			keyName = "notifyInPvpSafezone",
+			name = "Notify even when you are in a PvP safezone",
+			description = "Receive player spawn notifications even when you are in a PvP safezone such as a bank",
+			section = playerSpawnNotificationsSection
+	)
+	default boolean notifyInPvpSafezone()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+			position = 34,
+			keyName = "notifyOnlyAttackablePlayers",
+			name = "Notify only on players in your combat range",
+			description = "Only receive player spawn notifications for players in your combat range",
+			section = playerSpawnNotificationsSection
+	)
+	default boolean notifyOnlyAttackablePlayers()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+			position = 35,
+			keyName = "notifyFriendsSpawning",
+			name = "Notify when a friend spawns",
+			description = "Receive player spawn notifcations for friends spawning",
+			section = playerSpawnNotificationsSection
+	)
+	default boolean notifyFriendsSpawning()
+	{
+		return true;
```

Comment:
I think having this off as a default is better because not everyone uses the spawn notifications for friend finder functionality.

```suggestion
		return false;
```